scripts/prepdocs.py

```diff
@@ -46,12 +49,15 @@ async def get_vision_key(credential: AsyncTokenCredential) -> Optional[str]:
         exit(1)
 
 
-async def setup_file_strategy(credential: AsyncTokenCredential, args: Any) -> FileStrategy:
+async def setup_file_strategy(credential: AsyncTokenCredential, args: Any) -> Strategy:
     storage_creds = credential if is_key_empty(args.storagekey) else args.storagekey
     blob_manager = BlobManager(
         endpoint=f"https://{args.storageaccount}.blob.core.windows.net",
         container=args.container,
+        account=args.storageaccount,
         credential=storage_creds,
+        resourceGroup=args.storageresourcegroup,
+        subscriptionId=args.subscriptionId,
```

Comment:
Interesting, the new feature requires specifying those 3 parameters?

Reply:
Right to generted the managedidentity connection string for blobl